datalad/distributed/tests/test_create_sibling_ria.py

```diff
@@ -136,7 +136,9 @@ def _test_create_store(host, base_path, ds_path, clone_path):
     res = ds.create_sibling_ria("ria+ssh://test-store:", "datastore",
                                 recursive=True, existing='reconfigure')
     eq_(len(res), 3)
-    assert_result_count(res, 3, status='ok', action="create-sibling-ria")
+    assert_result_count(res, 1, path=str(ds.pathobj), status='ok', action="create-sibling-ria")
+    assert_result_count(res, 1, path=str(subds.pathobj), status='ok', action="create-sibling-ria")
+    assert_result_count(res, 1, path=str(subds2.pathobj), status='ok', action="create-sibling-ria")
```

Comment:
Just FTR: `str(ds.pathobj)` == `ds.path`

Reply:
Well, it's built the other way around. We still have a lot of `os.path` based things going on, so I'm not too sure what happens on windows. In my estimation reducing the direct use `Dataset.path` might help in eventually getting to a point where we really rely on `pathlib` instead.